src/plugins/vis_type_timeseries/public/application/components/aggs/percentile_rank/percentile_rank.tsx

```diff
@@ -121,17 +126,36 @@ export const PercentileRankAgg = (props: PercentileRankAggProps) => {
             />
           </EuiFormRow>
         </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer />
-      {model.values && (
-        <PercentileRankValues
-          disableAdd={isTablePanel}
-          disableDelete={isTablePanel}
-          showOnlyLastRow={isTablePanel}
-          model={model.values}
-          onChange={handlePercentileRankValuesChange}
-        />
-      )}
+
+        <EuiFlexItem>
+          <EuiFormRow
+            label={
+              <FormattedMessage
+                id="visTypeTimeseries.percentileRank.values"
+                defaultMessage="Values"
+              />
+            }
+          >
+            <>
+              {model.values && (
+                <PercentileRankValues
+                  disableAdd={isTablePanel}
```

Comment:
I get a warning for the key here when I add multiple Values
<img width="728" alt="Screenshot 2020-09-24 at 4 15 39 PM" src="https://user-images.githubusercontent.com/17003240/94149952-387a5900-fe81-11ea-8acc-58fe1296f9ce.png">
I don't reproduce it in master.

Reply:
@stratoula you are not right, on master I see the same issue. But it's not a reason to don't solve it. Will be fixed in that PR

Reply:
fixed

Reply:
My mistake @alexwizp, thank you so much for fixing this